### PR TITLE
Fix autostart file symlink

### LINKED ISSUE #76: Auto switch to dark theme stopped working
### What Happened?

I've noticed that dark theme doesn't turn on automatically in the evening. I have to set it up manually. And it also doesn't switch back to light theme in the morning. I do select "from sunset to sunrise" option every time.

### Steps to Reproduce

In the day time:
1. Set the dark theme
2. Turn on from sunset to sunrise
3. Nothing happens

In the night time:
0. Check the setting: there are  default theme and "from sunset to sunrise" that are chosen
1. System do not switches automatically to the dark theme

### Expected Behavior

Everything works according to the selected setting.

### OS Version

7.x (Horus)

### Software Version

Latest release (I have run all updates)

### Log Output

```shell
Sorry, don't know how to provide more information, but can cooperate 🙄
```

### Hardware Info

_No response_

### PR DESCRIPTION
**Pull request by lenemter**
Fixes #76

Closes #75

### CODE REVIEW

**Comment by danirabbit** [comment]
Probably should also incorporate #75 while here?

File: data/meson.build [hunk 1]
```diff
@@ -1,12 +1,12 @@
 install_data(
     'autostart.desktop',
-    install_dir: join_paths(get_option('datadir'), 'applications'),
+    install_dir: join_paths(get_option('prefix'), get_option('datadir'), 'applications'),
```
Comment (danirabbit) [inline data/meson.build:3]: looks like prefix and datadir are already defined variables?
```suggestion
    install_dir: prefix / data_dir / 'applications',
```

**Comment by lenemter** [comment]
@Marukesu with `install_symlink` the stable and unstable CI fail. Any idea why?

**Comment by Marukesu** [comment]
it's seem we are triggering mesonbuild/meson#10176, the fix is present in 0.61.5.

**Comment by bobby285271** [comment]
> I think this PR does the exact same thing as https://github.com/elementary/settings-daemon/pull/75 except checking is_absolute, which is false in both elementary OS and NixOS.

[According to docs](https://mesonbuild.com/Builtin-options.html), when the prefix is /usr, sysconfdir defaults to /etc, I think in this case is_absolute is true?

**Comment by bobby285271** [comment]
> I think this PR does the exact same thing as https://github.com/elementary/settings-daemon/pull/75 except checking is_absolute, which is false in both elementary OS and NixOS.

[According to docs](https://mesonbuild.com/Builtin-options.html), when the prefix is /usr, sysconfdir defaults to /etc, I think in this case is_absolute is true?

**Comment by lenemter** [comment]
@bobby285271 Thanks for clarification, I added changes from #75 here.

File: meson.build [hunk 2]
```diff
@@ -2,6 +2,7 @@ project('io.elementary.settings-daemon',
     'c', 'vala',
     version: '1.3.0',
     license: 'GPL3',
+    meson_version: '>=0.61.0'
```
Comment (bobby285271) [inline meson.build:5]: No longer needs 0.61?

Comment (lenemter) [reply meson.build:5]: I'm not too sure. There is https://github.com/elementary/triage/issues/175 which recomends to set minimum meson version to 0.61.2. I think we should follow it.

Comment (bobby285271) [reply meson.build:5]: Hmmm, I think at least 0.58 features are used here so the distro affected in my mind are CentOS 8 Stream, openSUSE Leap 15.4, Slackware 15.0, I think I can give ACK if no Pantheon users are there... :woman_shrugging:

**Review by bobby285271 — APPROVED** [review]

File: meson.build (hunk 2 above)
Comment (lenemter) [reply meson.build]: Thanks, I lowered required version to 0.58 just in case.

**Comment by lenemter** [comment]
Double checked and autostart file is correctly installed.

**Pull request merged**